data/core/src/commonMain/kotlin/com/tunjid/heron/data/network/models/QuotedPostConversions.kt

```diff
@@ -16,23 +16,38 @@
 
 package com.tunjid.heron.data.network.models
 
+import app.bsky.embed.RecordViewRecord
```

Comment:
I think all the changes in this file can be reverted. The logic in it is already defined.